src/pages/RegisterPage.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div class="register">
+    <CredentialsSection
+      v-if="currentPanel === 'CredentialsSection'"
+      @goToNextStep="goToNextStep"
+    ></CredentialsSection>
```

Comment:
Prefer self-closing tags when no content goes inside 😄 
```suggestion
    <CredentialsSection
      v-if="currentPanel === 'CredentialsSection'"
      @goToNextStep="goToNextStep"
    />
```